### PR TITLE
📝 Add documentation about integration tests

### LINKED ISSUE #430: [DOCS] add example on how to package and launch tests from test files in poetry
The issue is described in [this](https://stackoverflow.com/questions/73490143/how-do-i-include-and-install-test-files-in-a-wheel-and-deploy-to-databricks) StackOverflow question. 

* dbx version used: 0.7.0
* Databricks Runtime version: N/A

### PR DESCRIPTION
**Pull request by renardeinside**
## Proposed changes

Describe the big picture of your changes here to communicate to the maintainers. If it fixes a bug or resolves a feature request, please provide a link to that issue.

Closes #430 

## Types of changes

What types of changes does your code introduce to dbx?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation Update (if none of the other choices apply)

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: docs/guides/python/integration_tests.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+!!! hint
+
+    For Notebook-based workflows, take a look at the [nutter project](https://github.com/microsoft/nutter).
+
+
+## :octicons-file-code-24:  Preparing the configuration
+
+* Add an entrypoint file to the `tests/entrypoint.py`
+
+```python title="tests/netrypoint.py"
```
Comment (theKashyap) [inline docs/guides/python/integration_tests.md:14]: `netrypoint` -> `entrypoint`

File: docs/guides/python/integration_tests.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+import pytest
+
+if __name__ == '__main__':
+    pytest.main(sys.argv[1:])
+```
+
+* Add your tests to the `tests/integration` folder
+* Configure the special test-oriented `workflow`:
+
+```yaml title="conf/deployment.yml"
```
Comment (theKashyap) [inline docs/guides/python/integration_tests.md:26]: Don't you need other parts you described in the SO answer?

```
custom:
  basic-cluster-props: &basic-cluster-props
    spark_version: "10.4.x-cpu-ml-scala2.12"

  basic-static-cluster: &basic-static-cluster
    new_cluster:
      <<: *basic-cluster-props
      num_workers: 1
      node_type_id: "{{some-node-type}}"

build:
  commands:
    - "poetry build -f wheel" #safe to use inside poetry venv
```

File: docs/guides/python/integration_tests.md (hunk 1 above)
Comment (renardeinside) [reply docs/guides/python/integration_tests.md:14]: hi @theKashyap , 
this one has been fixed in later PRs, now it points to the correct one, please take a look at the up-to-date docs page.

File: docs/guides/python/integration_tests.md (hunk 2 above)
Comment (renardeinside) [reply docs/guides/python/integration_tests.md:26]: This one has been fixed in #446 and released in 0.7.4

**Pull request merged**